### PR TITLE
Regles de gestions des status

### PR DESCRIPTION
**Pull request by ydiarra**
- Create modal to accept a bubble
- Create modal to go back to online status
- Apply management rules for new status

### CODE REVIEW

**Comment by gparant** [comment]
Review for @ydiarra 

When a User is in the meeting, a rule implemented is that we cannot change the status and it works 💪

I think that we need to show information at the user throughout the tooltip to show what the status can't be changed during a meeting. Examples of fun interaction should be "Have a good meeting 🤓".

Screenshot 2024-03-06 at 08 27 09

**Review by gparant — COMMENTED** [review]
Screenshot 2024-03-06 at 08 27 09

File: play/src/front/Components/ActionBar/AvailabilityStatus/AvailabilityStatus.svelte [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    };
+
+    let listProps: AvailabilityStatusListPropsInterface = {
+        currentStatus: $availabilityStatusStore,
+        listStatusTitle: $LL.actionbar.listStatusTitle.enable(),
+        statusInformations: getStatusInformation(statusToShow),
+    };
+
+    let buttonProps: AvailabilityStatusPropsInterface = {
+        currentPlayerName: localStorage.getItem("playerName") || "",
```
Comment (moufmouf) [inline play/src/front/Components/ActionBar/AvailabilityStatus/AvailabilityStatus.svelte:48]: use localUserStore.getUserName instead

File: play/src/front/Rules/StatusRules/StatusStrategy/BusyStatusStrategy.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    private lastNotificationPermissionRequestMoreThanTwoWeeks = (d1: Date): boolean => {
+        const diffTime = Math.abs(new Date().getTime() - d1.getTime());
+        const diffDays = diffTime / (1000 * 60 * 60 * 24);
+        const diffWeeks = Math.floor(diffDays / 7);
+        return diffWeeks >= 2;
+    };
+
+    private showNotificationPermissionModal = () => {
+        const localStoragelastNotificationPermissionRequest: string | null = localStorage.getItem(
```
Comment (moufmouf) [inline play/src/front/Rules/StatusRules/StatusStrategy/BusyStatusStrategy.ts:45]: move localStorage access to localUserStore

**Pull request merged**